Library/Formula/xsd.rb

```diff
@@ -0,0 +1,49 @@
+class Xsd < Formula
+  desc "XML Data Binding for C++"
+  homepage "http://www.codesynthesis.com/products/xsd/"
+  url "http://www.codesynthesis.com/download/xsd/4.0/macosx/i686/xsd-4.0.0-i686-macosx.tar.bz2"
```

Comment:
On 2 June 2015 at 11:14, Mike McQuaid <notifications@github.com> wrote:

> In Library/Formula/xsd.rb
> <https://github.com/Homebrew/homebrew/pull/40274#discussion_r31504657>:
>
> > @@ -0,0 +1,49 @@
> > +class Xsd < Formula
> > +  desc "XML Data Binding for C++"
> > +  homepage "http://www.codesynthesis.com/products/xsd/"
> > +  url "http://www.codesynthesis.com/download/xsd/4.0/macosx/i686/xsd-4.0.0-i686-macosx.tar.bz2"
>
> Sorry, just noticed this: is this a compiled binary? We should build this
> from source or put it in homebrew/binary
>
It is, indeed. I'd say just put this into whatever section it needs to be
(or should I file a pull request elsewhere?) and, when I find time for it,
I will look into compiling it if it seems preferable for the project.




> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/40274/files#r31504657>.
>